code/modules/clothing/gloves/insulated.dm

```diff
@@ -12,6 +12,10 @@
 	resistance_flags = NONE
 	custom_price = PAYCHECK_CREW * 10
 	custom_premium_price = PAYCHECK_COMMAND * 6
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
```

Comment:
Your PR body does not justify why you would also add this. Rubber electrician gloves are not notably cold-resistant.